doc/content/validation/free_flow/heat_transfer/flat_plate/flat_plate.md

```diff
@@ -0,0 +1,142 @@
+The boundary conditions for the momentum and pressure equations are the following:
+
+!table id=tab:momentum-pressure-bc caption=Momentum and pressure boundary conditions.
+| Boundary | Boundary condition |
+| --- | --- |
+| +Fluid inlet+ | Fixed velocity, $u_{in}=13~\frac{m}{s}$ |
+| +Fluid top+ | Free stream |
+| +Fluid outlet+      | Outlet pressure, $p_{out}=1.035\times10^5~Pa$ |
+| +Fluid bottom and solid top+ | No-slip |
+
+The boundary conditions for the fluid and solid energy equations are the following:
+
+!table id=tab:energy-bc caption=Solid and fluid energy boundary conditions.
+| Boundary | Boundary condition |
+| --- | --- |
+| +Fluid inlet+ | Fixed temperature, $T_{in}=1000~K$ |
+| +Fluid top+ | Insulated |
+| +Fluid outlet+ | Outlet enthalpy |
+| +Fluid bottom before solid+ | Insulated |
+| +Solid top+ | Conjugate heat transfer |
+| +Solid bottom+ | Fixed temperature $T_b=600~K$ |
+| +Solid sides+ | Insulated |
+
+Material properties used are given in [tab:matprops].
+
+!table id=tab:matprops caption=Material properties used in the CHT benchmark case.
+| Region | Density, +$\rho$+ $\left(\frac{kg}{m^3}\right)$ | Specific Heat, +$c_p$+ $\left(\frac{J}{kg\cdot K}\right)$ | Conductivity, +$k$+ $\left(\frac{W}{m\cdot K}\right)$ | Dynamic Viscosity, +$\mu$+ $\left(Pa\cdot s\right)$ |
+| --- | --- | --- | --- | --- |
+| Fluid (air) | 0.3525 | 1142.6 | 0.06808 | 3.95$\times 10^{-5}$ |
+| Solid (fictional) | Not used | Not used | 0.2876 | Not used |
+
+## `OpenPronghorn` Model
+
+The mesh is generated using the MOOSE mesh generator system and includes both solid and fluid regions. A
+representative section of the mesh is shown in +[fig:mesh]+ below. The cell density near the fluid–solid
+interface is refined to capture strong thermal gradients. The refinement is achieved by the introduction of
+mesh bias parameters.
+
+!media media/validation/free_flow/heat_transfer/flat_plate/mesh.png style=width:70%;margin-left:auto;margin-right:auto;text-align:center; id=fig:mesh caption=Mesh showing the fluid and solid regions in the 2D CHT channel.
+
+Altogether, the mesh has 125 divisions horizontally in the entrance region and 250 in the fluid region
+above the plate. The solid plate has 80 while the fluid domain 160 vertical divisions. Altogether,
+the mesh consists of 80,000 quad cells. A mesh sensitivity has been carried out and the included model
+provides closely converged results with the least amount of cells.
+
+The simulation employs a +SIMPLE solver+ to obtain the steady state solution. Second order geometric average interpolation has been
+used for the advection discretization, while the stress and conduction operators have been
+discretized using an arithmetic average discretization. The mesh is orthogonal, so no correction terms were used.
+The input file for this case is embedded below.
+
+!listing /validation/free_flow/heat_transfer/flat_plate/cht_rob-rob.i
+
+## Results
+
+Two key validation metrics are examined:
```

Comment:
a validation metric would be an error on something right

maybe "Two key physical quantities are being considered to build validation metrics " ?